### PR TITLE
roll back PROMETHEUS_VERSION to v1.4.1

### LINKED ISSUE #128: Update to prometheus 1.5.2 make prometheus UI unusable
Update to Prometheus 1.5.2 make Prometheus Graph UI unusable
While the DB itself works fine (for Grafana), the add graph button throws Handlebar Exception
https://github.com/prometheus/prometheus/issues/2399

### PR DESCRIPTION
**Pull request by tzach**
fix #128

### CODE REVIEW

**Comment by tzach** [comment]
@amnonh please review and validate

File: start-all.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ VERSIONS=$DEFAULT_VERSION
 usage="$(basename "$0") [-h] [-d Prometheus data-dir] [-v comma seperated versions] [-g grafana port ] [ -p prometheus port ] -- starts Grafana and Prometheus Docker instances"
 
 GRAFANA_VERSION=4.1.1
-PROMETHEUS_VERSION=v1.5.2
+PROMETHEUS_VERSION=v1.4.1
```
Comment (amnonh) [inline start-all.sh:8]: I think 1.5.1 works fine

Comment (tzach) [reply start-all.sh:8]: Did you test it? It did not works for me
Make sure to do a hard reload between test, otherwise you are using the cached JS

**Comment by amnonh** [comment]
On Mon, Feb 20, 2017 at 10:05 AM, Tzach Livyatan <notifications@github.com>
wrote:

> *@tzach* commented on this pull request.
> ------------------------------
>
> In start-all.sh
> <https://github.com/scylladb/scylla-grafana-monitoring/pull/129#discussion_r101963058>
> :
>
> > @@ -5,7 +5,7 @@ VERSIONS=$DEFAULT_VERSION
>  usage="$(basename "$0") [-h] [-d Prometheus data-dir] [-v comma seperated versions] [-g grafana port ] [ -p prometheus port ] -- starts Grafana and Prometheus Docker instances"
>
>  GRAFANA_VERSION=4.1.1
> -PROMETHEUS_VERSION=v1.5.2
> +PROMETHEUS_VERSION=v1.4.1
>
> Did you test it? It did not works for me
> Make sure to do a hard reload between test, otherwise you are using the
> cached JS
>
I thougt it did, but I'm not sure with the javascript anymore, when it
strat working, it works for all versions.

**Comment by tzach** [comment]
> I thought it did, but I'm not sure with the javascript anymore, when it
> strat working, it works for all versions.

Same here. The issues does not related to the version.

**Pull request closed without merging**